prow/github/client.go

```diff
@@ -185,6 +198,25 @@ func (c *Client) Throttle(hourlyTokens, burst int) {
 	c.throttle.throttle = throttle
 }
 
+// RoundTrip is an implementation for net/http RoundTripper interface.
+// Its being used to refresh the token from a token source.
+func (ct *cacherTransport) RoundTrip(req *http.Request) (resp *http.Response, err error) {
+	_, err = ct.Base.Source.Token()
```

Comment:
Im not sure why you are loading the token, but not using the result. The oauth2 transport already loads the token each RoundTrip so this function shouldn't need to load the token itself. https://github.com/golang/oauth2/blob/master/transport.go#L35